backend/github.js

```diff
@@ -0,0 +1,20 @@
+var exports = module.exports = {};
+var request = require("request");
+
+exports.retrieveOrgs = function (accessToken, callback) {
+  var organisations = [];
```

Comment:
Variables can be properly used in functional scope. It can be declared in callback function of request module.